index.html

```diff
@@ -229,9 +338,9 @@ <h2 class="icons-header">LET'S CONNECT</h2>
           </ul>
         </div>
         <ul class="icons-lang">
-          <li><img src="./assets/images/js.png" alt="js" />Javascript</li>
-          <li><img src="./assets/images/html.png" alt="html" />HTML</li>
-          <li><img src="./assets/images/css.png" alt="Ellipse" />CSS</li>
+          <li class="li-icons"><img src="./assets/images/js.png" alt="js" />Javascript</li>
+          <li class="li-icons"><img src="./assets/images/html.png" alt="html" />HTML</li>
+          <li class="li-icons"><img src="./assets/images/css.png" alt="Ellipse" />CSS</li>
         </ul>
         <div>
           <ul class="lang">
```

Comment:
- [ ] As requested by the first reviewer, kindly fix the link to the arrow icon in the technologies section so that the icon can display properly. Consider changing the src attribute of the img tag to `src="./assets/images/arrow.png"`
![image](https://user-images.githubusercontent.com/94551484/221346168-7bb49bb8-b395-4881-a3ce-fb4a0821532d.png)